src/clients/update_proposal.rb

```diff
@@ -213,6 +213,8 @@ def main
           @ret["warning"] = @warning_message
           @ret["warning_level"] = product_warning["warning_level"] || :warning
         end
+        # save the solver test case with details for later debugging
+        Pkg.CreateSolverTestCase("/var/log/YaST2/solver-upgrade-proposal") if @ret["warning"]
```

Comment:
do you test if save_y2logs will include it and also that after upgrade it will be copied to system?

Reply:
Yes, `save_y2logs` saves that, I have verified that.

But I haven't checked whether it's copied to `/mnt`, but if you are able to finish the upgrade then that means you were somehow able to solve the dependency issues in the end.

I'll check it later when doing some real migration...